src/pages/LoginAndRegister/RegisterSocial.vue

```diff
@@ -48,10 +48,10 @@
 					v-model="newAcctTerms"
 					:validation="$v.newAcctTerms"
 				>
-					I have read and agree to the Kiva
+					I have read and agree to the {{ !passwordless ? 'Kiva' : '' }}
 					<a href="/legal/terms" target="_blank">Terms of Use</a>
 					and
-					<a href="/legal/privacy" target="_blank">Privacy Policy</a>
+					<a href="/legal/privacy" target="_blank">Privacy Policy</a> {{ !passwordless ? '' : '(required)' }}
```

Comment:
I don't think we should remove this b/c the field is required and we need to state that.

Reply:
Also, this is an addition. I think it should be global.

Reply:
done